README.md

```diff
@@ -23,6 +23,7 @@ see the Istio [contribution guidelines](https://github.com/istio/community/blob/
 - [Testing document content](#testing-document-content)
 - [Multi-language support](#multi-language-support)
 - [Regular maintenance](#regular-maintenance)
+- [Website documentation](./WEBSITE.md)
```

Comment:
This is all documentation for the "website," so I'm not sure that's the best name. Would a better place for this be here? https://istio.io/latest/about/contribute/

Reply:
What happening to all the documentation at https://istio.io/latest/about/contribute/? How should contributors find that going forward?

Reply:
I would propose moving that content to GitHub.  Contributor docs for code are on GitHub; contributor docs for the web should be there too. If the team wanted to keep this on the site, it would need to move under Docs.

Reply:
Perhaps we could call the link on README.md 'Changing and deploying the website'? I also like having it in GitHub. Many projects have instructions to change and set up a local environment in their readmes/similar files as the last section of the README.